packages/legend-query-builder/src/components/QueryBuilder.tsx

```diff
@@ -150,6 +153,20 @@ const QueryBuilderStatusBar = observer(
           >
             <HackerIcon />
           </button>
+          <button
```

Comment:
This is fine, but I also have to warn you of the potential consequence of this, in the embedded query builder, we now have the button to hide virtual assistant, that would make this button showing up twice in the UI (including the one in Legend Studio status bar)